src/workerd/jsg/util.h

```diff
@@ -394,4 +394,23 @@ inline bool isFinite(double value) {
   return !(kj::isNaN(value) || value == kj::inf() || value == -kj::inf());
 }
 
+// ======================================================================================
+
+class Lock;
+
+v8::MaybeLocal<v8::String> newExternalOneByteString(Lock& js, kj::ArrayPtr<const char> buf);
+v8::MaybeLocal<v8::String> newExternalTwoByteString(Lock& js, kj::ArrayPtr<uint16_t> buf);
```

Comment:
You will need this to be `ArrayPtr<const uint16_t>` if you want to be able to pass built-in arrays.